resources/web/wwi/protos/ProtoCamera.proto

```diff
@@ -4,16 +4,26 @@ EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/project
 
 PROTO ProtoCamera [
 field     SFFloat                                        fieldOfView            0.785398
-field     SFInt32                                        width                  64   
+field     SFInt32                                        width                  64
 field     SFInt32                                        height                 64
 field     SFFloat                                        near                   0.01
 field     SFFloat                                        far                    0.0
+field     SFFloat                                        minRange               0.01
+field     SFFloat                                        maxRange               1
+field     SFFloat                                        lidarFieldOfView            1.5708
+field     SFFloat                                        lidarVerticalFieldOfView    0.2
+field     SFInt32                                        lidarNumberOfLayers         4
+field     SFInt32                                        lidarHorizontalResolution        512
+field     SFFloat                                        lidarMinRange               0.01
+field     SFFloat                                        lidarMaxRange               1
+field     SFFloat                                        lidarTiltAngle              0.0
```

Comment:
Alignment could be improved here (lots of useless spaces, and non-aligned values).

Reply:
Yes, but note that this proto (as all protos in the wwi/protos folder are just for the internal testing of `feature-web-proto`) and will be removed before merging `feature-web-proto` in `develop`